src/utils/MetaboliteSpectralMatcher.cpp

```diff
@@ -91,106 +85,104 @@ class TOPPMetaboliteSpectralMatcher :
         public TOPPBase
 {
 public:
-    TOPPMetaboliteSpectralMatcher() :
-        TOPPBase("MetaboliteSpectralMatcher", "Find potential HMDB ids within the given mass error window.", false)
-    {
-    }
+  TOPPMetaboliteSpectralMatcher() :
+      TOPPBase("MetaboliteSpectralMatcher", "Find potential HMDB ids within the given mass error window.", false)
+  {
+  }
 
 protected:
 
-    void registerOptionsAndFlags_()
-    {
-        registerInputFile_("in", "<file>", "", "mzML file");
-        setValidFormats_("in", ListUtils::create<String>("mzML"));
-        registerOutputFile_("out", "<file>", "", "mzTab file");
-        setValidFormats_("out", ListUtils::create<String>("csv"));
-
-        // addEmptyLine_();
-        // addText_("Parameters for the accurate mass search can be given in the 'algorithm' part of INI file.");
-        registerSubsection_("algorithm", "Algorithm parameters section");
-    }
-
-    Param getSubsectionDefaults_(const String& /*section*/) const
+  void registerOptionsAndFlags_()
+  {
+    registerInputFile_("in", "<file>", "", "mzML file");
+    setValidFormats_("in", ListUtils::create<String>("mzML"));
+    registerInputFile_("database", "<file>", "CHEMISTRY/MetaboliteSpectralDB.mzML", "mzML file", false);
```

Comment:
Yes, that is true. Currently, we don't have a place where we collect this information so its hard to give an URL here. I think I will leave it as is right now until we found a proper place to put the spectral library (and also have someone taking care of updating it)